stdpopsim/cli.py

```diff
@@ -96,7 +95,7 @@ def write_output(ts, args):
     ts.dump(args.output)
 
 
-def write_citations(chromosome, model):
+def write_citations(contig, model):
```

Comment:
this is an awesome, thoughtful touch.

Reply:
Citations be important --- we want model writers to be motivated to contribute stuff. If their paper won't get cited ever again after it gets implemented in stdpopsim then there isn't much motivation to contribute...